dev-vcs/cvs/cvs-1.12.12-r15.ebuild

```diff
@@ -0,0 +1,103 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit autotools pam toolchain-funcs
+
+DESCRIPTION="Concurrent Versions System - source code revision control tools"
+HOMEPAGE="http://cvs.nongnu.org/"
+
+SRC_URI="mirror://gnu/non-gnu/cvs/source/feature/${PV}/${P}.tar.bz2
+	doc? ( mirror://gnu/non-gnu/cvs/source/feature/${PV}/cederqvist-${PV}.html.tar.bz2
+		mirror://gnu/non-gnu/cvs/source/feature/${PV}/cederqvist-${PV}.pdf
+		mirror://gnu/non-gnu/cvs/source/feature/${PV}/cederqvist-${PV}.ps )"
+
+LICENSE="GPL-2 LGPL-2"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~loong ~m68k ~mips ~ppc ~ppc64 ~riscv ~s390 ~sparc ~x86 ~x64-cygwin ~amd64-linux ~x86-linux ~ppc-macos ~sparc-solaris ~sparc64-solaris ~x64-solaris ~x86-solaris"
+
+IUSE="crypt doc kerberos nls pam server"
+RESTRICT="test"
+
+DEPEND=">=sys-libs/zlib-1.1.4
+	kerberos? ( virtual/krb5 )
+	pam? ( sys-libs/pam )
+	server? ( virtual/libcrypt:= )"
+RDEPEND="${DEPEND}"
+
+src_unpack() {
+	unpack ${P}.tar.bz2
+	use doc && unpack cederqvist-${PV}.html.tar.bz2
+}
+
+PATCHES=(
+	"${FILESDIR}"/${P}-cvsbug-tmpfix.patch
+	"${FILESDIR}"/${P}-openat.patch
+	"${FILESDIR}"/${P}-block-requests.patch
+	"${FILESDIR}"/${P}-cvs-gnulib-vasnprintf.patch
+	# "${FILESDIR}"/${P}-install-sh.patch
+	"${FILESDIR}"/${P}-hash-nameclash.patch # for AIX
+	"${FILESDIR}"/${P}-getdelim.patch # 314791
+	"${FILESDIR}"/${PN}-1.12.12-rcs2log-coreutils.patch # 144114
+	"${FILESDIR}"/${P}-mktime-x32.patch # 395641
+	"${FILESDIR}"/${P}-fix-massive-leak.patch
+	"${FILESDIR}"/${P}-mktime-configure.patch #220040 #570208
+	"${FILESDIR}"/${P}-CVE-2012-0804.patch
+	"${FILESDIR}"/${P}-format-security.patch
+	"${FILESDIR}"/${P}-musl.patch
+	"${FILESDIR}"/${P}-CVE-2017-12836-commandinjection.patch
+	"${FILESDIR}"/${P}-fix-implicit-function-declarations.patch
+	)
+DOCS=( BUGS ChangeLog{,.zoo} DEVEL-CVS FAQ HACKING MINOR-BUGS NEWS \
+	PROJECTS README TESTS TODO )
+
+src_prepare() {
+	eapply "${FILESDIR}"/${P}-fix-autoconf.patch
```

Comment:
Could I ask why this specific patch is here instead of PATCHES?

Reply:
You sure can. 
During source prepare I wanted to fix autoconf first, THEN run eautoreconf and only THEN apply the other patches. The other patches mangle with configure (and they still apply after autoconf miraculously except the one for install-sh) so their changes would be overwritten.

Reply:
Ty 😄

Reply:
I reconsidered this